corehq/apps/hqwebapp/static/hqwebapp/js/select2_handler_v4.js

```diff
@@ -16,6 +16,7 @@ hqDefine("hqwebapp/js/select2_handler_v4", [
         var self = {};
         self.fieldName = options.fieldName;
         self.placeholder = options.placeholder;
+        self.allowClear = options.allowClear || true;
```

Comment:
this will always be true. I think that you want something like:

```
self.allowClear = options.allowClear !== undefined ? options.allowClear : true
```

Reply:
Oops!. Sorry for this. slipped from mind in hurry. Next time will be more careful. 
Done correcting